elasticsearch/process_unites_legales.py

```diff
@@ -78,20 +83,73 @@ def process_unites_legales(chunk_unites_legales_sqlite):
             unite_legale["sigle"],
             unite_legale["prenom"],
         )
+
         # Replace missing values with 0
         unite_legale_processed["nombre_etablissements_ouverts"] = (
             0
             if unite_legale_processed["nombre_etablissements_ouverts"] is None
             else unite_legale_processed["nombre_etablissements_ouverts"]
         )
+
+        # Dirigeants
+        unite_legale_processed["liste_dirigeants"] = []
+        unite_legale_processed["dirigeants_pp"] = json.loads(
+            unite_legale["dirigeants_pp"]
+        )
+
```

Comment:
Can we had here a deduplication of `unite_legale_processed["dirigeants_pp"]` and `unite_legale_processed["dirigeants_pm"]` ?
Due to constraint in processing inpi data, it may be multiple time the same object in inpi.db. At this step of processing (after querying sirene.db) we can deduplicates in an efficient way dirigeants.

Reply:
Done !